src/foam/u2/crunch/wizardflow/AutoSaveWizardletsAgent.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.crunch.wizardflow',
+  name: 'AutoSaveWizardletsAgent',
+  documentation: `
+    Binds listeners which automatically save wizardlets when they are modified.
+  `,
+
+  imports: [
+    'wizardlets'
+  ],
+
+  implements: [
+    'foam.core.ContextAgent'
+  ],
+
+  methods: [
+    async function execute() {
```

Comment:
Why is this async?

Reply:
This is done to match the interface. `foam.util.async.Sequence` expects this method to return a Promise.